virtualpdu/pdu/__init__.py

```diff
@@ -19,6 +19,43 @@
 
 logger = logging.getLogger(__name__)
 
+# A textual description of the entity. This value should include the full
+# name and version identification of the system's hardware type, software
+# operating-system, and networking software. It is mandatory that this only
+# contain printable ASCII characters.
+sysDescr = (1, 3, 6, 1, 2, 1, 1, 1)
+
+# The vendor's authoritative identification of the network management
+# subsystem contained in the entity. This value is allocated within the SMI
+# enterprises subtree (1.3.6.1.4.1) and provides an easy and unambiguous
+# means for determining 'what kind of box' is being managed. For example, if
+# vendor `Flintstones, Inc.' was assigned the subtree 1.3.6.1.4.1.4242, it
+# could assign the identifier 1.3.6.1.4.1.4242.1.1 to its 'Fred Router'.
+sysObjectID = (1, 3, 6, 1, 2, 1, 1, 2)
```

Comment:
After discussion, we should remove the descriptions because they are a little misleading and the person can query the oid-info to get information on those OID.